lib/ProductOpener/Tags.pm

```diff
@@ -1058,7 +1058,7 @@ sub get_file_from_cache ($source, $target) {
 # Change this version string if you want to force the taxonomies to be rebuilt
 # e.g. if the taxonomy building algorithm or configuration has changed
 # This needs to be done also when the unaccenting parameters for languages set in Config.pm are changed
-my $BUILD_TAGS_VERSION = "20240329 - better taxonomy errors handling";
+my $BUILD_TAGS_VERSION = "20240403 - fix issue with additives.properties.txt not loaded + circular_parent check";
```

Comment:
you should not need it if you touch ingredients.txt

This is only needed when the code change.

Reply:
```suggestion
my $BUILD_TAGS_VERSION = "20240329 - better taxonomy errors handling";
```

Reply:
ok, let's merge asap :-)